public/components/health-check/lib/check-kibana-settings.ts

```diff
@@ -24,13 +26,15 @@ async function getKibanaSettings(): Promise<responseKbnSettings> {
 
 async function checkMetafieldSetting({settings}: responseKbnSettings) {
   const { metaFields } = settings;
-  return !!metaFields && !!metaFields.userValue.length;
+  const isEqual = _.isEqual(WAZUH_METAFIELDS, metaFields)
+  return !!metaFields && !isEqual;
 }
 
 async function updateMetaFieldsSetting(isModified:boolean) {
+  console.log("UPDATE",!isModified)
   return !isModified && await GenericRequest.request(
     'POST',
     '/api/kibana/settings',
-    {"changes":{"metaFields":[]}}
+    {"changes":{"metaFields":["_source","_index"]}}
```

Comment:
Why hardcode the metafields? You could use a constant as:
```
WAZUH_KIBANA_SETTING_METAFIELDS = ['_source', '_index']
```

Add the contant to the `constants.ts` file and adapt the comparation with `_isEqual` (not use userValue in the constant)